### DISCUSSION TITLE
virustotal detection

### DISCUSSION THREAD

**Discussion by Alex-test-mod** [Babyhamsta/Aimmy#101] [General]
https://www.virustotal.com/gui/file/69b656552158342e8098892c1c14bd645cee9f2b1cfac5c82c00ed6802d9bb4b/detection

**Comment by operator436** [comment]
Just found this trojan as part of a file for Aimmy an AI cheat for Hell Let Loose on github.
[AimmyV2.1.5.zip](http://aimmyv2.1.5.zip/) This is a Backdoor.MSIL.gkhl a malicious tool to give backdoor to your computer. They can then control your computer remotely however they want.
Cheats aren't free, they are trojans to attack our government, infrastructure, military targets, hold ransom attacks on our fortune 500 companies and much worse than you can imagine. Once these are installed they use your powerful gaming system to attack, blackmail and destroy assets. Maybe even use your computer to host child images. And you might get held responsible!
By installing these cheats for games, you are helping enemies of the free world be attacked. GetGud ... stop cheating!
PS
The author will lie their *ss off to tell you it's a false positive. Don't all criminals lie before the boof you in the *ss?
to see this for yourself go to [virustotal.com](http://virustotal.com/) drop the zip file on the website to have every single anti-virus in the known universe against it.

**Reply by operator436** [reply]
I'd also like to add, that you as a gamer with your high end computers and fast internet are the main target of these thousand/s man digital armies of China, Russian, Pakistan, North Korea, criminal enterprises, etc.
So you aren't just risking your computer that you think, meh I don't bank on it, meh I don't care, meh I just want to win because I am a lama who can't play because I'm a pot smoking low IQ retard. At least you can be responsible for everyone else who works for a living in the free world.  Or help the enemy and your children will someday be slaves to these tyrannical crazy ass governments.

**Comment by jasonacox-sam** [comment]
Thanks for the report @marcg1! I've opened #300 to track this bug. The root cause is that the v1r Owner API login uses a `tesla://` custom URL scheme redirect that browsers can't handle. I'm working on a fix that switches to the same native WebView PKCE flow already used by cloud mode setup.
